givitsite/friendreq/models.py

```diff
@@ -11,18 +11,14 @@
 ]
 
 ITEM_CHOICES = [
-    ('table','table'),
-    ('bed','bed'),
-    ('TV','TV'),
-    ('washing machine','washing machine'),
-    ('fridge','fridge'),
-    ('oven','oven'),
-    ('microwave oven','microwave oven'),
-    ('chair','chair'),
-    ('computer screen','computer screen'),
-    ('keyboard','keyboard'),
-    ('laptop','laptop'),
-    ('closet','closet'), 
```

Comment:
If I get the Django docs right, the first value in the tuple is what is stored in the DB and the 2nd is what gets shown in the UI - therefore I'd expect the 1st to be in English or even be a number, and the 2nd be in Hebrew

Generally I;d keep non-ascii characters out of the code and use some localization features like translation files in order to show then in the UI

Reply:
@ifireball @rotem-benzvi actually, for this stage, we want the values in the DB, will be in Hebrew because the Gatherer app will use them on searching (this is not for the UI). 

anyway, I will find another solution to avoid non-ASCII chars in the code

Reply:
I think having the Gatherer access the DB directly may be a bad design decision - it means you always have to worry about how the DB looks internally instead of letting the ORM "worry" about it.

I suggest you do one of the following:
1. Merge the Gatherer app into the Django code base so it can import and use the ORM too
2. Have the Gatherer app talk to Django via some HTTP-based API instead of to the DB directly.

Those are big changes though, and you have a milestone on Tuesday, so LMK if you want to merge things as they are and do a big fix later (And open issues to track it!). This might make fixing things later a bit harder though.

Reply:
The Gatherer app is a Django app, and it uses ORM.
For now - we will use IDs for the items, it is better than Hebrew and allows us to sync with the Gatherer.

fixed in the latest commit
@ifireball @rotem-benzvi

Reply:
those still seem to be strings rather then numbers (ints) but I guess its better then Hebrew text...

Think if you want to fix it now or later...